cosmos/operators/local.py

```diff
@@ -252,6 +256,67 @@ def store_compiled_sql(self, tmp_project_dir: str, context: Context, session: Se
         else:
             self.log.info("Warning: ti is of type TaskInstancePydantic. Cannot update template_fields.")
 
+    @staticmethod
+    def _configure_remote_target_path() -> tuple[Path, str] | tuple[None, None]:
+        """Configure the remote target path if it is provided."""
+        if not remote_target_path:
+            return None, None
+
+        _configured_target_path = None
+
+        target_path_str = str(remote_target_path)
+
+        remote_conn_id = remote_target_path_conn_id
+        if not remote_conn_id:
+            target_path_schema = target_path_str.split("://")[0]
+            remote_conn_id = FILE_SCHEME_AIRFLOW_DEFAULT_CONN_ID_MAP.get(target_path_schema, None)  # type: ignore[assignment]
+        if remote_conn_id is None:
```

Comment:
Perhaps this could be in the same line as 262

Reply:
When, the user has not specified the remote conn_id as part of settings, the remote_conn_id could be inferred on just the line above this line from the map that we keep, and hence, I am deferring to check it here to give another try at identifying the remote_conn_id (i.e. the default conn_id kept in Airflow provider based on the scheme)